### PR TITLE
feat(services): update all modal

### PR DESCRIPTION
**Pull request by bdebon**
# What does this PR do?

[> Link to the JIRA ticket](https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-634)

CleanShot 2023-01-12 at 11 21 20@2x

---

### Global

- [ ] This PR does not introduce any breaking change
- [ ] This PR introduces breaking change(s) and has been labeled as such
- [ ] I have found someone to review this PR and pinged him

### Store

- [ ] This PR introduces new store changes

### NX

- [ ] I have run the dep-graph locally and made sure the tree was clean i.e no circular dependencies
- [ ] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`

### Clean Code

- [ ] I made sure the code is type safe (no any)
- [ ] I have included a feature flag on my feature, if applicable

### CODE REVIEW

**Comment by evoxmusic** [comment]
A preview environment was automatically created via Qovery.
Click on the link below to follow its deployment and use it.
:point_right: [[PR] staging - feat(services): update all modal - 2023-01-12T10:21:51Z](https://console.qovery.com/platform/organization/141c07c8-0dd9-4623-983b-3fdd61867255/projects/977bd069-20f7-4b24-bd95-28b44174cf43/environments/7a5eb808-0fee-4c75-9f51-1d5babe5cbc2/applications)

 Another comment will be posted when deployments are terminated

**Comment by evoxmusic** [comment]
Your preview environment has been successfully deployed !
Click on the link below to open your service:
:point_right: [console](https://z7a5eb808-z27af4d57-gtw.za8ad0659.bool.sh)
:point_right: [storybook](https://z7a5eb808-z99ecb39b-gtw.za8ad0659.bool.sh)

File: libs/shared/console-shared/src/lib/environment-buttons-actions/ui/environment-buttons-actions.tsx [hunk 1]
```diff
@@ -167,23 +168,23 @@ export function EnvironmentButtonsActions(props: EnvironmentButtonsActionsProps)
...
-      //     },
-      //   }
-      // bottomItems.push(updateAllButton)
+      const updateAllButton = {
+        name: 'Update all services',
+        contentLeft: <Icon name={IconAwesomeEnum.ROTATE} className="text-sm text-brand-400" />,
+        onClick: (e: ClickEvent) => {
+          e.syntheticEvent.preventDefault()
+          //projectId,
+          //environmentId: environment.id,
```
Comment (RemiBonnet) [inline libs/shared/console-shared/src/lib/environment-buttons-actions/ui/environment-buttons-actions.tsx:177]: You can remove this one

BTW on this file, you have a warning here 
Capture d’écran 2023-01-17 à 15 16 59

File: libs/shared/console-shared/src/lib/update-all-modal/feature/update-all-modal-feature.tsx [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+  useEffect(() => {
+    dispatch(fetchApplications({ environmentId }))
+  }, [dispatch, environmentId])
+
+  useEffect(() => {
+    if (appsLoadingStatus === 'loaded') {
+      applications?.forEach((application) => {
+        if (isApplication(application) || isGitJob(application)) {
+          if (!application.commits || application.commits.loadingStatus === 'not loaded') {
```
Comment (RemiBonnet) [inline libs/shared/console-shared/src/lib/update-all-modal/feature/update-all-modal-feature.tsx:63]: Can you merge the two "if" ?

**Review by RemiBonnet — COMMENTED** [review]
LGTM 👍 some comments on the code and two improvements: 

- Can you add same placeholder that update service by commit ?
- Can add an button to select all ? 
Capture d’écran 2023-01-17 à 15 22 38

**Pull request merged**